fixtures/VisibleMethods/VisibleMethods.java

```diff
@@ -0,0 +1,88 @@
+package VisibleMethods;
+
+class VisibleMethods {
+	class Complex2 {
+		String n = "";
+
+		public void n1() {
+			n2("Hello world");
+		}
+
+		private void n2(String in) {
+			n = in;
+			System.out.println(n);
+		}
+	}
+
+	private int a = 10;
+	private int b = 20;
+	private double c = 30.5;
+	private String d = "Name";
+
+	private void m1() {
+		int x = 10;
+
+		a = x + 1;
+		b = a * 20;
+	}
+
+	private void m2() {
+		int a = 10;
+		a = 20;
+
+		int bbb = this.a * 20;
+	}
+
+	public void m3() {
+		this.a = 20;
+		int x = this.a * 10;
+	}
+
+	void m4() {
```

Comment:
so, this one is not considered visible then?

Reply:
See the comment above, regarding visible methods.